csv_detective/detect_fields/other/float_field/__init__.py

```diff
@@ -0,0 +1,19 @@
+import re
+
+PROPORTION = 1
+
+def _is(val):
+    '''Detects floats'''
+    regex = r'[-+]?[ ]?([0-9]*\.[0-9]+|[0-9]+)'
+    return bool(re.match(regex, val))
+
+if __name__ == '__main__':
+    print(_is('500,0'))
+
+    print(_is('500'))
+
+    print(_is('500.0'))
+
+    print(_is('500b0')) # TODO fix this as this is not good
+
+    print(_is('b 500.0'))
```

Comment:
I would prefer for folder name : "float" like "booleen" example. We are on "Detect fields" folder so there is no ambiguity for me if we rename it only "float"

Reply:
ok will do

Reply:
renamed to 'floats' to avoid confusion with python 'float' usages